envs/linc/jupyterhub-overrides.yaml

```diff
@@ -0,0 +1,100 @@
+hub:
+  config:
+    Authenticator:
+      admin_users:
+        - "asmacdo"
+        - "dandibot"
+        - "satra"
+        - "aaron"
```

Comment:
```suggestion
        - "aaronkanzer"
```

Reply:
Thanks @asmacdo I bumped these values in my most recent commit.